src/components/with-parallax/index.js

```diff
@@ -43,6 +37,28 @@ const withParallax = function( WrappedComponent ) {
 			const scrollContainer = document.getElementsByClassName( 'edit-post-layout__content' )[ 0 ];
 			window.addEventListener( 'resize', this.updateHandler );
 			scrollContainer.addEventListener( 'scroll', this.updateHandler );
+
+			this.observers = findParents( this.container, '.wp-block' ).map( block => {
+				const observer = new MutationObserver( movements => {
+					movements.forEach( movement => {
+						if ( 'style' === movement.attributeName ) {
+							if ( movement.oldValue.includes( 'transform: translate3d' ) ) {
```

Comment:
Would we get in trouble if we get rid of this condition? I feel like there could also be other style changes that would require `updateDimensions`. Also we would also be covered if `translate3d` turns into `translateY` at some point.

Reply:
If we remove the conditions it would update on any attribute change, classes, data sets and all style changes. For example, it will trigger an update on `is-hovered`... even if nothing changes.

Reply:
I was referring only to the latter condition comparing the old value of the style tag but since this is just an assumption I'll merge the pull request and wait to see what the future brings us. Congrats and thank you for your first contribution @acosmin

Reply:
Thanks 😄 you could probably remove that condition, there aren't many styles applied on that element...